operators/admin/olm-upgrading-operators.adoc

```diff
@@ -8,5 +8,22 @@ toc::[]
 
 As a cluster administrator, you can upgrade Operators that have been previously installed using Operator Lifecycle Manager (OLM) on your {product-title} cluster.
 
+[IMPORTANT]
+====
+//The upgrade from OpenShift sandboxed containers 1.2 to 1.3 does not upgrade all 1.2 resources. The monitor pods (metrics) continue to run 1.2 images, and `KataConfig` is not be updated to the new version. You must upgrade the images manually once the upgrade is complete. You will also need to manually patch `KataConfig`, or delete and recreate it, to apply the new functionality and bug fixes to the Operator.
+
+
+The upgrade from OpenShift sandboxed containers 1.2 to 1.3 does not upgrade the `KataConfig` custom resource. As a result, the monitor pods (metrics) are not restarted and continue to run version 1.2 images.
+
+Following the upgrade, manually upgrade the monitor image by either editing the `KataConfig` YAML in the web console, or running the following CLI command:
++
+[source,terminal]
+----
+$ oc patch kataconfig example-kataconfig --type merge --patch '{"spec":{"kataMonitorImage":"registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.3.0"}}'
+----
+
+Next, manually patch `KataConfig`, or delete and recreate it, to apply the new functionality and bug fixes to the Operator.
```

Comment:
The `KataConfig` tracks the deployment of the monitor pods and the kata rpm, not the operator:
- new features and bug fixes in the operator code itself are available right after the upgrade
- manually patching the `KataConfig` is needed to get new features and bug fixes for the monitor pods
- deleting / re-creating the `KataConfig` is needed to get new features and bug fixes for kata

Reply:
In progress.